packs/data/spells.db/scorching-ray.json

```diff
@@ -58,6 +42,76 @@
         "materials": {
             "value": ""
         },
+        "overlays": {
+            "hqiwae0wgehycnex": {
+                "_id": "hqiwae0wgehycnex",
+                "data": {
+                    "damage": {
+                        "value": {
+                            "d80radfbikwjogmr": {
+                                "applyMod": false,
+                                "type": {
+                                    "categories": [],
+                                    "subtype": "",
+                                    "value": "fire"
+                                },
+                                "value": "4d6"
+                            }
+                        }
+                    },
+                    "heightenedLevel": {},
```

Comment:
What is this doing here? That shouldn't exist anymore in top level spell data. Is something slipping it in?

Reply:
I've just added a new heightened entry on the variants. Seems like it was added there. I'll take a look.